### DISCUSSION TITLE
Add supported or unsupported platform information for some templates, especially for ARM host

### DISCUSSION THREAD

**Discussion by nvroot** [coollabsio/coolify#4263] [Feature Requests]
Coolify runs smoothly on ARM, and most templates work without issues. However, some official templates do not support ARM and consequently do not function properly. Examples include Bookstack (supports but doesn't works), Trigger.dev, Cyberchef and some others too. It would be helpful to include an informational warning when attempting to install these templates on an ARM host.

**Comment by peaklabs-dev** [comment]
They do work on ARM though they just run the AMD image version through Docker no?

**Comment by nvroot** [comment]
@peaklabs-dev 
1) BookStack does have an ARM image available, but it does not work. Attempts to build it manually have also failed, and even when the ARM image was successfully built, it still did not work.
2) Trigger.dev's provider and coordinator modules does not support ARM, as indicated [Provider](https://ghcr.io/triggerdotdev/provider/docker) [Coordinator](https://ghcr.io/triggerdotdev/coordinator) 
3) CyberChef does not have an ARM image available and attempts to compile it have also been unsuccessful.

**Comment by nvroot** [comment]
@peaklabs-dev looks like you are right, from different sources it's possible to get working samples.

**Comment by nvroot** [comment]
@peaklabs-dev You are absolutely right, all the mentioned applications (some with little modifications) fully run on ARM architecture -  I have personally tested and confirmed they work. However, they are not officially supported. For example, I added CyberChef in the templates using official image which only support x86. Adding third-party container images creates potential security risks, especially when maintainers are not well-known, if almost add not be better inform about that?
It's not clear how quickly security updates are added to builds by unknown maintainers or whether harmful code could be introduced. It would be helpful to clearly state that these images are officially supported only for the x86 architecture. For example, additionally on documentation page also would be noted possibily of make arm image if required..
